acceptance/main.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2024 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package main
+
+import "github.com/redpanda-data/redpanda-operator/harpoon/tablegenerator"
+
+var providers = []string{"eks", "aks", "gke", "kind"}
```

Comment:
Ah, should have mentioned this in the other PR but can we start with k3d here instead of kind? We're going to need the ability to add and remove K8s nodes and kind does not have the ability to do so.

I saw the note about using kind to start but it should really just be a difference for changing a CLI invocation, no?